scripts/appveyor/config.ps1

```diff
@@ -25,7 +25,7 @@ $config = @{
       database = "master";
       requestTimeout = 25000;
       cryptoCredentialsDetails = @{
-        ciphers = "RC4-MD5"
```

Comment:
The `RC4-MD5` cipher is no longer available in default Node.js 10.x builds, as it's totally insecure. As I'm not really sure which other ciphers are available and compatible with SQL Server on Appveyor, I just took the "old backward compatibility" cipher list as recommended by https://wiki.mozilla.org/Security/Server_Side_TLS#Old_backward_compatibility.